tensorflow/tools/ci_build/Dockerfile.cpu

```diff
@@ -8,6 +8,7 @@ RUN /install/install_bootstrap_deb_packages.sh
 RUN add-apt-repository -y ppa:openjdk-r/ppa
 RUN /install/install_deb_packages.sh
 RUN /install/install_bazel.sh
+RUN /install/install_pip.sh
```

Comment:
1) See comment below. No need to put this in a separate file. Just do pip install here. 
2) The same change needs to be made in Dockerfile.gpu in the same directory.